beetsplug/beatport.py

```diff
@@ -234,7 +234,14 @@ def __init__(self, data: JSONDict):
         if "artists" in data:
             self.artists = [(x["id"], str(x["name"])) for x in data["artists"]]
         if "genres" in data:
-            self.genres = [str(x["name"]) for x in data["genres"]]
+            genre_list = [str(x["name"]) for x in data["genres"]]
```

Comment:
Given my comment above, I expect `beatport` plugin to simply make sure that it _always_ provides `genres` list.